controllers/sts.go

```diff
@@ -142,7 +143,7 @@ func createDSStatefulSet(ds *directoryv1alpha1.DirectoryService, sts *apps.State
 						{
 							Name:            "init",
 							Image:           ds.Spec.Image,
-							ImagePullPolicy: v1.PullAlways, // todo: for testing this is good. Remove later?
+							ImagePullPolicy: v1.PullIfNotPresent,
```

Comment:
As per https://kubernetes.io/docs/concepts/containers/images/#updating-images , you can force image pulls by leaving the default setting: `IfNotPresent` + using the tag `latest`. `Always` is not required.